CONTRIBUTING.md

```diff
@@ -96,6 +96,19 @@ Then from the root directory, you can run:
 
   Note: this is not through turbo, so you'll have to build turbo yourself first.
 
+- Updating Integration Tests
+
+  ```
+  pnpm -- turbo run build --filter=cli
+  pnpm test:interactive -F turborepo-tests-integration
```

Comment:
I'm not seeing what the `-F` flag does

```console
~/src/github.com/vercel/turbo/turborepo-tests/integration$ pnpm prysk --help
Running /home/tr/src/github.com/vercel/turbo/turborepo-tests/integration/.cram_env/bin/prysk --shell=bash   --help
usage: prysk [OPTIONS] TESTS...

positional arguments:
  TESTS                 Path(s) to the tests to be executed

options:
  -h, --help            show this help message and exit
  -V, --version         show program's version number and exit
  -q, --quiet           don't print diffs (default: False)
  -v, --verbose         show filenames and test status (default: False)
  -i, --interactive     interactively merge changed test output (default: False)
  -d, --debug           write script output directly to the terminal (default: False)
  -y, --yes             answer yes to all questions (default: False)
  -n, --no              answer no to all questions (default: False)
  -E, --preserve-env    don't reset common environment variables (default: False)
  --keep-tmpdir         keep temporary directories (default: False)
  --shell PATH          shell to use for running tests (default: /bin/sh)
  --shell-opts OPTS     arguments to invoke shell with (default: None)
  --indent NUM          number of spaces to use for indentation (default: 2)
  --color {always,never,auto}
                        Mode which shall be used for coloring the output (default: auto)
  --xunit-file PATH     path to write xUnit XML output (default: None)
  --dos2unix            convert DOS/Windows line endings to UNIX line endings (default: False)
  ```

Reply:
should have said first:

```console
~/src/github.com/vercel/turbo$ pnpm test:interactive -F turborepo-tests-integration
 ERR_PNPM_RECURSIVE_EXEC_FIRST_FAIL  Command "test:interactive" not found

Did you mean "pnpm test:integration"?
```

What you nave here doesn't wok for me from root, did you mean:

```suggestion
  pnpm --filter turborepo-tests-integration test:interactive
```